packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -324,7 +324,7 @@ function NetworkListboxPlusBalancesContainer({
   children: React.ReactNode
 }) {
   return (
-    <div className="flex flex-row flex-nowrap items-center justify-between gap-1 whitespace-nowrap">
+    <div className="flex flex-col items-start sm:flex-row sm:items-center justify-between gap-1 gap-y-2.5 whitespace-nowrap">
```

Comment:
@mistersimon Sorry for the delay on reviewing this. Could you also run `yarn prettier:format` in the terminal, so that it sorts tailwind classes according to our rules. Thanks!

Reply:
Hey @bartlomiej94  - Don't apologise for the delay, completely understand. 

Good spot on the formatting, thats been done now - should be in the right order.